plugins/emoji/plugin.js

```diff
@@ -194,7 +194,7 @@
 					output.push( this.createEmojiListBlock() );
 					output.push( this.createStatusBar() );
 
-					return '<div class="cke_emoji-inner_panel">' + output.join( '' ) + '</div>';
+					return CKEDITOR.tools.htmlSafeByReview('<div class="cke_emoji-inner_panel">' + output.join( '' ) + '</div>', 'Template created using safe internal values');
```

Comment:
Just for clarity, let's rephrase since they're not using "template" here.

Reply:
Rephrased to 'Content created using..." for clarity.